src/foam/nanos/crunch/crunchtest/CapabilityTest.java

```diff
@@ -201,6 +202,50 @@ public void testCapabilityAuthService(X x) {
 
   }
 
+  public void testCapabilityExpires(X x) {
+
+    User user = new User();
+    user.setFirstName("TestCapabilityExpireUser");
+    user.setId(9001);
+    user = (User) userDAO.put_(x, user);
+
+    AuthService auth = (AuthService) x.get("auth");
```

Comment:
How do we know that this is actually decorated with CapabilityAuthService? It's commented out in the service script for the "auth" service but I can't find anywhere that it's added or mocked out for this test.

Reply:
None of the tests work until the decorator is enabled in services.jrl